pkg/services/featuremgmt/toggles_gen.json

```diff
@@ -3064,6 +3079,21 @@
         "hideFromDocs": true
       }
     },
+    {
+      "metadata": {
+        "name": "unifiedStorageSearch",
+        "resourceVersion": "1726771421439",
+        "creationTimestamp": "2024-09-19T18:43:41Z",
+        "deletionTimestamp": "2024-10-11T15:54:21Z"
```

Comment:
Same, it's weird to have the deletion timestamp now, why wasn't it here in the first place :open_mouth:

Reply:
Ok fixed with a merge on main.